django_extensions/mongodb/fields/__init__.py

```diff
@@ -145,6 +146,20 @@ def create_slug(self, model_instance, add):
             next += 1
         return slug
 
+    def get_slug_fields(self, model_instance, lookup_value):
+        lookup_value_path = lookup_value.split(LOOKUP_SEP)
+        attr = model_instance
+        for elem in lookup_value_path:
+            try:
+                attr = getattr(attr, elem)
+            except AttributeError:
+                return None
```

Comment:
Did you intentionally kept the `return None` here ?
Thinking might be best to also throw exception in case of Mongo.